platform/windows/display_server_windows.cpp

```diff
@@ -247,8 +247,13 @@ class FileDialogEventHandler : public IFileDialogEvents, public IFileDialogContr
 	// IUnknown methods
 	HRESULT STDMETHODCALLTYPE QueryInterface(REFIID riid, void **ppv) {
 		static const QITAB qit[] = {
+#ifdef __MINGW32__
+			{ &__uuidof(IFileDialogEvents), static_cast<int>(OFFSETOFCLASS(IFileDialogEvents, FileDialogEventHandler)) },
```

Comment:
This should be OK, but we might want something like this, since `QITAB` seems to be defined with a different type in different versions.

```suggestion
			{ &__uuidof(IFileDialogEvents), static_cast<decltype(qit[0].dwOffset)>(OFFSETOFCLASS(IFileDialogEvents, FileDialogEventHandler)) },
```